### PR TITLE
feat: add initial post-install page to the wiki

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

File: wiki/index.html [hunk 1]
```diff
@@ -57,8 +57,8 @@ <h1>The distribution</h1>
             &bull; <a href="quick-start.html">Quick Start Guide (Post-install)</a> <br /> <br >
 
             <b>Tools used in Rhino Linux:</b> <br />
-            &bull; <a href="rhino-pkg.html">rhino-pkg</a> <br /> 
-            &bull; <a href="rhino-setup.html">Rhino Setup Wizard</a> <br />
+            &bull; <a href="https://github.com/rhino-linux/rhino-pkg">rhino-pkg</a> <br />
```
Comment (httpllamaz) [inline wiki/index.html:60]: This is meant to link to our wiki page about it, not the github.

File: wiki/index.html [hunk 2]
```diff
@@ -57,8 +57,8 @@ <h1>The distribution</h1>
             &bull; <a href="quick-start.html">Quick Start Guide (Post-install)</a> <br /> <br >
 
             <b>Tools used in Rhino Linux:</b> <br />
-            &bull; <a href="rhino-pkg.html">rhino-pkg</a> <br /> 
-            &bull; <a href="rhino-setup.html">Rhino Setup Wizard</a> <br />
+            &bull; <a href="https://github.com/rhino-linux/rhino-pkg">rhino-pkg</a> <br /> 
+            &bull; <a href="https://github.com/rhino-linux/rhino-setup">Rhino Setup Wizard</a> <br />
```
Comment (httpllamaz) [inline wiki/index.html:61]: This is meant to link to our wiki page about it, not the github.

File: wiki/index.html [hunk 3]
```diff
@@ -71,14 +71,14 @@ <h1>Development</h1>
 
         <p>
             The official documentation and portal for the development of Rolling Rhino Remix. It is recommended if you are contributing to Rhino Linux to join our Discord Server or Matrix Space: <a href="https://rhinolinux.org/contact">https://rhinolinux.org/contact</a>. <br /><br />
-            &bull; <a href="https://github.com/rhino-linux/os">Disk image generation</a>
+            &bull; <a href="https://github.com/rhino-linux/os">Disk image generation repository</a>
```
Comment (httpllamaz) [inline wiki/index.html:74]: This is good

Comment (cat-master21) [reply wiki/index.html:74]: So I can resolve this specificly?

File: wiki/quick-start.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        <footer>
+            <p>
+                <a href="https://github.com/rhino-linux">Source Code</a> | <a href="../contact">Contact & Social</a>  | <a href="https://liberapay.com/Rolling-Rhino-Remix/">Donate</a>  | <a href="../brand">Branding</a> 
+            </p>
+        </footer>
+
+    </div>
+
+</body>
+</html>
```
Comment (httpllamaz) [inline wiki/quick-start.html:86]: The whole page needs rewriting to actually be a step-by-step guide, I can do it tomorrow.

**Review by httpllamaz — CHANGES_REQUESTED** [review]

File: wiki/quick-start.html (hunk 4 above)
Comment (cat-master21) [reply wiki/quick-start.html:86]: So you think it's good for merge, but needs to be rewritten.

**Pull request closed without merging**